docs/troubleshooting.md

```diff
@@ -0,0 +1,249 @@
+certsLastUpdated: null
+certsRotateAt: null
+conditions:
+- lastTransitionTime: "2019-11-21T14:45:58Z"
+  lastUpdateTime: "2019-11-21T14:45:58Z"
+  message: csv in namespace with no operatorgroups
+  phase: Failed
+  reason: NoOperatorGroup
+lastTransitionTime: "2019-11-21T14:45:58Z"
+lastUpdateTime: "2019-11-21T14:46:40Z"
+message: csv in namespace with no operatorgroups
+phase: Failed
+reason: NoOperatorGroup
+```
+
+>Note: It is possible that the Status is missing, which suggests that the OLM operator is encountering an issue when processing the `CSV` in a very early stage. You should respond by reviewing the logs of the OLM operator.
+
+From the information within the `status.reason` and `status.message` fields, we now know that the `CSV` failed to install because it has been deployed in a namespace without an [OperatorGroup](./operator-scoping.md).
+
+Let's create an `OperatorGroup` in the `my-catalogsource-namespace namespace` by applying the following yaml:
+```sh
+$ cat operatorgroup.yaml
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: my-operators
+  namespace: my-catalogsource-namespace
+```
+
+Now we should inspect the status of the `CSV` again:
+```
+kubectl -n my-catalogsource-namespace get csvs
+NAME                        DISPLAY               VERSION   REPLACES                    PHASE
+prometheusoperator.0.32.0   Prometheus Operator   0.32.0    prometheusoperator.0.27.0   Failed
+```
+
+The `CSV` is still in the failed state. Let's reinspect the `reason` and `message` within the status block of the `CSV` with the following commands:
+```sh
+$ kubectl -n my-catalogsource-namespace get csv prometheusoperator.0.32.0 -o yaml | yq r - status.reason
+reason: UnsupportedOperatorGroup
+
+$ kubectl -n my-catalogsource-namespace get csv prometheusoperator.0.32.0 -o yaml | yq r - status.message
+message: AllNamespaces InstallModeType not supported, cannot configure to watch all namespaces
+```
+
+From the output above we can derrive that the `CSV` is failing to install because it has been deployed in a namespace that includes an `OperatorGroup` that whose installMode is not supported by the `CSV`.
+
+As a cluster admin, I can fix this install issue by updating the `OperatorGroup` to support one of the `CSV's` supported installmodes. I can get a list of supported installModes with the following commands:
+
+```sh
+$ kubectl -n my-catalogsource-namespace get csv prometheusoperator.0.32.0 -o yaml | yq r - spec.installModes
+- supported: true
+  type: OwnNamespace
+- supported: true
+  type: SingleNamespace
+- supported: false
+  type: MultiNamespace
+- supported: false
+  type: AllNamespaces
+```
+
+From here, we can follow the steps outlined in [operator-scoping.md](./operator-scoping.md) to create or modify an OperatorGroup that the operator supports. In this scenario, we will update the `OperatorGroup` to match the `OwnNamespace` install mode:
```

Comment:
Does it make more sense to have olm book redirecting to another part of OLM book instead of md file, since this is a website?
[operator-scoping](https://operator-framework.github.io/olm-book/docs/operator-scoping)

Reply:
Just a heads up: we're switching to relative links in the docs here.